tests/unit/components/file-picker-test.js

```diff
@@ -90,4 +90,22 @@ test('it shows file input', function(assert) {
   this.render();
 
   assert.equal(component.$('input:hidden').length, 0);
-});
\ No newline at end of file
+});
+
+test('it rejects improper filetypes', function(assert) {
+  assert.expect(2);
+
+  const component = this.subject({
+    accept: '.jpg,.jpeg',
+    multiple: true
+  });
+
+  const files = [
+    { filename: 'goodfile.jpg' },
+    { filename: 'good_file.jpeg' },
+    { filename: 'badfile.html' }
+  ];
+
+  assert.strictEqual(component._invalidExtension(files), true);
+  assert.strictEqual(component._invalidExtension(files.slice(0, 2)), false);
+});
```

Comment:
Can you add some tests that check if handleFiles works properly with the option turned on and off?
Something like [that](https://github.com/timmcanty/ember-cli-file-picker/blob/validate-with-accept-field/tests/unit/components/file-picker-test.js#L40-70)